editor/filesystem_dock.cpp

```diff
@@ -1429,14 +1430,17 @@ void FileSystemDock::_files_list_rmb_select(int p_item, const Vector2 &p_pos) {
 
 	file_options->clear();
 	file_options->set_size(Size2(1, 1));
-	if (all_files && filenames.size() > 0) {
-		file_options->add_item(TTR("Open"), FILE_OPEN);
-		if (all_files_scenes) {
+	if (all_files) {
+
+		if (all_files_scenes && filenames.size() >= 1) {
+			file_options->add_item(TTR("Open"), FILE_OPEN);
```

Comment:
Maybe this one should be renamed to "Open Scene(s)"?

Reply:
Mmm...I agree with you.
![b](https://user-images.githubusercontent.com/24988459/33597330-04efbb62-d9da-11e7-90bb-15b623c55755.png)